variables.tf

```diff
@@ -89,7 +89,7 @@ variable "eks_worker_ami_name_filter" {
 variable "eks_worker_ami_name_regex" {
   type        = string
   description = "A regex string to apply to the AMI list returned by AWS"
-  default     = "^amazon-eks-node-[1-9,.]+-v[0-9]{8}$"
+  default     = "^amazon-eks-node-[0-9,.]+-v[0-9]{8}$"
```

Comment:
is it worth the trouble to ensure that the first digit in the version is still 1-9? 

For example:
```
"^amazon-eks-node-[1-9][0-9,.]+-v[0-9]{8}$"
```